recipes/yasm/all/test_package/conanfile.py

```diff
@@ -0,0 +1,8 @@
+from conans import ConanFile
+import os
+
+
+class TestPackage(ConanFile):
+
+    def test(self):
+        self.run("yasm --help")
```

Comment:
I'm curious if run_environment is required here not, I mean, what is the chance of finding yasm from system first.

Reply:
it doesn't use any shared libraries, executable is standalone, but I may add it, just for consistency